listenbrainz/webserver/templates/user/resetlatestimportts.html

```diff
@@ -0,0 +1,25 @@
+{% extends 'base.html' %}
+
+{% block title %}Reset import timestamp- ListenBrainz{% endblock %}
+
+{% block content %}
+  <h3 class="page-title">Reset Import Timestamp</h3>
+  <p>Are you sure you want to reset your import timestamp? </p>
+
+  <p>
+     Our last.fm importer keeps track of the listens you've imported and then only imports
```

Comment:
This text is good! I think we need to bring some of this back to the page where we show the button in the first palace.